test/built-ins/Object/getOwnPropertyDescriptors/tamper-with-global-object.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2016 Jordan Harband. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: >
+    Object.getOwnPropertyDescriptors should not have its behavior impacted by modifications to the global property Object
+es7id: pending
+author: Jordan Harband
+---*/
+
+function fakeObject() {
+    $ERROR('The overriden version of Object was called!');
+}
+fakeObject.getOwnPropertyDescriptors = Object.getOwnPropertyDescriptors;
+fakeObject.keys = Object.keys;
+
+var global = Function('return this;')();
```

Comment:
This logic is encapsulated in [the `fnGlobalObject.js` helper function](https://github.com/tc39/test262/blob/fbad34959ffedb02c5b8c040e0f992a6fc006f36/harness/fnGlobalObject.js)